README.md

```diff
@@ -440,17 +441,38 @@ combineLatest: [false, true, true, true]
 
 ------
 
+### FilterMany
+Filters element of a publisher collection into a new publisher collection.
+```swift
+let intArrayPublisher = PassthroughSubject<[Int], Never>()
+
+intArrayPublisher
+.filterMany { $0.isMultiple(of: 2) }
+.sink(receiveValue: { print($0) })
+
+intArrayPublisher.send([10, 1, 2, 4, 3, 8])
+```
+
+#### Output:
+
+```
+none
+[10, 1, 2, 4, 8]
```

Comment:
That 1 shouldn't be there in the example's output I think?

Reply:
yup seems like this should be 10 , 2, 4 , 8 , right ?

Reply:
🙈

Reply:
Fixed